project2/styles.css

```
@@ -0,0 +1,201 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    color: #686868;
+    font-size: 18px;
+}
+
+section {
+    background-color: #faf4e3;
+    max-width: 640px;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+header {
+    background-image: url(images/shrimps_2.jpg);
```

There's only one photo, so maybe consider renaming this?

---

Also, you want to keep your URLs inside quotes, like so:
```suggestion
    background-image: url("images/shrimps_2.jpg");
```

This is working as is, but this makes sure your background-image and filename will be properly displayed regardless of the system and server you're using for your web site.

---

Also, while it looks like you did a great job at resizing this image to be appropriate for the card width, you may have neglected to optimize the image.

[tinyPNG](https://tinypng.com/) says it can be optimized for a filesize savings of 80%. As you can imagine, on a site with many articles on a single page, those savings would add up, especially for mobile users.